shared-module/packages/common/src/components/UnstyledA.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "@emotion/styled"
+
+const UnstyledA = styled.a`
+  all: unset;
```

Comment:
Please don't use `all: unset;`.

It removes out *all* default browser styles, including useful ones like focus outlines -- which are important for keyboard navigation and accessibility.

Instead, just override the specific styles you don’t want, and leave the rest of the defaults alone.